codalab/lib/bundle_cli.py

```diff
@@ -1033,10 +1042,12 @@ def do_workers_command(self, args):
                     'cpus': '{}/{}'.format(worker['cpus_in_use'], worker['cpus']),
                     'gpus': '{}/{}'.format(worker['gpus_in_use'], worker['gpus']),
                     'memory_bytes': formatting.size_str(worker['memory_bytes']),
+                    'free_disk_bytes': formatting.size_str(worker['free_disk_bytes']),
```

Comment:
can you change the column headers to `memory` and `free_disk` since the values are already formatted, so the `bytes` is inaccurate?  Also will save some horizontal space.